.bintray.json.in

```diff
@@ -0,0 +1,28 @@
+{
+    "package": {
+        "name": "on-imagebuilder",
+        "repo": "debian", 
+        "subject": "rackhd",
+        "vcs_url": "https://github.com/RackHD/on-imagebuilder.git",
+        "licenses": ["Apache-2.0"]
+    },
+
+    "version": {
+        "name": "#REVISION#"
+    },
+
+    "files":
+        [
+            {
+                "includePattern": "deb/(.*)", "uploadPattern": "/dists/trusty/main/binary-amd64/$1", 
```

Comment:
if we specific trusty/main/amd64, why we still need to specific the folder /dists/trusty/main/binary-amd64 ?
say, can we just "uploadPattern": "$1" ?